pyproject.toml

```diff
@@ -15,7 +15,7 @@ dependencies = [
     "python-magic~=0.4.27",
     "python-dotenv==1.0.0",
     # LLM Triad
-    "unstract-adapters~=0.2.0",
+    "unstract-adapters~=0.2.1",
```

Comment:
Can you bump the SDK version to `0.10.1` as well @johnyrahul?

Reply:
NIT: Update the lock file as well if you can, if not I can take care of that along with my changes